### DISCUSSION TITLE
Is possible to enable API multiple interface IPs?

### DISCUSSION THREAD

**Discussion by dalamardruid** [bluenviron/mediamtx#1628] [Questions]
I would need to activate the HTTP API simultaneously on a static IP interface and on a dynamic IP interface

**Comment by Docteh** [comment]
What about the classic 0.0.0.0 listen address?

**Comment by aler9** [comment]
Hello, use 0.0.0.0 and tune the firewall to whitelist IPs.

[a comment by an automated account is not shown]
